Gemfile.lock

```diff
@@ -352,12 +352,14 @@ GEM
     zeitwerk (2.7.1)
 
 PLATFORMS
-  aarch64-linux
+  aarch64-linux-musl
+  aarch64-linux-gnu
```

Comment:
Minor, but when I run `bundle lock --add-platform` to add the new platforms, it adds them in alphabetical order. It probably doesn't make a difference, but should we follow that?

Reply:
Good spot - have updated!